browser/components/MarkdownPreview.js

```diff
@@ -895,6 +895,22 @@ export default class MarkdownPreview extends React.Component {
       this.setImgOnClickEventHelper(img, rect)
       imgObserver.observe(parentEl, config)
     }
+
+    const aList = markdownPreviewIframe.contentWindow.document.body.querySelectorAll('a')
+    for (const a of aList) {
+      // Removes a hash
+      const hashLink = a.hash.substr(1)
+      const fileRegExp = new RegExp('^file://')
+      const mainHtmlRegExp = new RegExp('/lib/main.html')
+      const isHeadingInMarkdownPreview = fileRegExp.test(a.href) && mainHtmlRegExp.test(a.href)
+      const targetEl = markdownPreviewIframe.contentWindow.document.getElementById(hashLink)
+      if (hashLink !== '' && isHeadingInMarkdownPreview && targetEl) {
+        const dataLine = targetEl.dataset.line
+        a.onclick = () => {
+          this.scrollTo(dataLine)
```

Comment:
Can you move this code into this function? Because we already have an existing link click listener.
https://github.com/BoostIO/Boostnote/blob/9a6ee9d2ef2d853b623d386c928088356ba4dbce/browser/components/MarkdownPreview.js#L1014-L1024

Reply:
I understood. I fixed it.

Since the link hash this time did not match the regular expression, the regular expression has been changed.
https://github.com/BoostIO/Boostnote/blob/857367efba4312b842b5ece6fd5e6912702090ff/browser/components/MarkdownPreview.js#L1029-L1030
e.g.
![FixRegExp](https://user-images.githubusercontent.com/11808736/57577793-cd152e80-74b9-11e9-8919-35facbed2a84.jpg)